src/AzureAuth/Commands/Info/CommandInfoResetDeviceID.cs

```diff
@@ -15,15 +15,18 @@ namespace Microsoft.Authentication.AzureAuth.Commands.Info
     [Command(Name = "reset-device-id", Description = "Reset your AzureAuth telemetry device identifier.")]
     public class CommandResetDeviceID
     {
+        private const string ApplicationName = "azureauth";

```

Comment:
This string was created in this PR, but never used? Did I miss?

Reply:
Good catch!